integration-tests/src/test/kotlin/spine/protodata/hello/NegativeCasesTest.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package spine.protodata.hello
+
+import io.spine.tools.gradle.testing.GradleProject
+import io.spine.tools.mc.java.gradle.McJavaTaskName
+import org.gradle.testkit.runner.UnexpectedBuildFailure
+import org.gradle.testkit.runner.internal.DefaultGradleRunner
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Assertions.fail
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import java.io.StringWriter
+
+/**
+ * Checks various cases where the `size` option is used incorrectly.
+ *
+ * The every test-case configures and runs the build of the test project
+ * in a separate Gradle process because any negative case raises
+ * the appropriate error and fails the build.
+ *
+ * Test-case is accepted when the specific error message is found in stderr
+ * stream of the failed build process.
+ */
+class `NegativeCasesTest should` {
```

Comment:
When describing a test case, one should describe the thing we test. I.e. if we test some job done by a `Humidifier`, then the test cases should be start with the following:
```
class `Humidified should` { ... }
```

In this case, we don't test `NegativeCasesTest`. We test our plugin.

Same goes to the filename.